src/core/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { ThemedBackground } from '../../utils/storybook';
+import Modal from './Modal';
+
+storiesOf('Modal', module)
+  // Kubera Chaos
+  .add('Kubera Chaos', () => (
+    <ThemedBackground platform="kubera-chaos">
+      <Modal
+        open={true}
+        onClose={() => console.log('clicked')}
```

Comment:
[Not an issue] onClose is basically optional since it toggles the modal open state to false. It can we used to perform extra operation here like logging on console but this is optional to be passed in as a prop.